java/memory/memory-netty/src/main/java/io/netty/buffer/UnsafeDirectLittleEndian.java

```diff
@@ -170,6 +166,66 @@ public ByteBuf setDouble(int index, double value) {
     return this;
   }
 
+  @Override
```

Comment:
I'd rather not add these if it's not necessary. I don't think the write* methods are even being used..